serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc

```diff
@@ -27,19 +27,19 @@ image::tooling/serverless-workflow-editor/swf-editor-vscode-extension-page.png[]
 .Procedure
 . You can install the VS Code extension for Serverless Workflow editor using one of the following methods:
 
-* *Go to Visual Studio Marketplace*: Go to link:{kogito_swf_editor_vscode_marketplace_url}[Serverless Workflow Editor] extension page in Visual Studio Marketplace and click *Install*.
+* *Go to Visual Studio Marketplace*: Go to link:{kogito_swf_editor_vscode_marketplace_url}[KIE Serverless Workflow Editor] extension page in Visual Studio Marketplace and click *Install*.
 +
 --
-.Serverless Workflow Editor in Visual Studio Marketplace
-image::tooling/serverless-workflow-editor/swf-editor-vscode-market-place.png[]
+.KIE Serverless Workflow Editor in Visual Studio Marketplace
+image::tooling/serverless-workflow-editor/swf-editor-vscode-marketplace.png[]
 --
 
 * *Download VSIX file*:
 +
 --
-.. Download the `vscode_extension_serverless_workflow_editor_VERSION.vsix` file from the link:{kie_tools_releases_page_url}[kie-tools] releases page in GitHub.
+.. Download the `serverless_workflow_vscode_extension_VERSION.vsix` file from the link:{kie_tools_releases_page_url}[kie-tools] releases page in GitHub.
```

Comment:
FYI, the vsix is not available on the release page :S

Reply:
Since 0.30.1

Reply:
Well, this is a side effect of kie-tools not rereleasing unchanged components every minor release. Maybe we change this guide to point to vscode marketplace?

Reply:
Yes, as Dominik mentioned, there might be a release where the vsix is not present. I've rephrased the text to avoid that issue. Thanks.